packages/artifacts/src/addresses.json

```diff
@@ -0,0 +1,6 @@
+{
+    "pluginRepo": {
+        "mainnet": "",
+        "polygon": ""
+    }
+}
```

Comment:
- What would be the structure of this json?
- Please also add the addresses that won't change like upgradable proxies